lib/normalizers/v1.js

```diff
@@ -42,16 +42,14 @@ _.assign(Builders.prototype, {
     auth: function (entityV1) {
         if (!entityV1) { return; }
         if (util.notLegacy(entityV1, 'auth') && entityV1.auth) {
-            if (entityV1.auth.type === 'noauth') { return; }
-
-            return util.sanitizeAuthArray(entityV1);
+            return (entityV1.auth.type === 'noauth') ? entityV1.auth : util.sanitizeAuthArray(entityV1);
         }
-        if (!entityV1.currentHelper || (entityV1.currentHelper === 'normal')) { return; }
+        if (!entityV1.currentHelper) { return; }
```

Comment:
Same as above

Reply:
This is the case where the auth properties are absent altogether, indicating that the auth for this entity can be overridden.